jenkins/onmetal_simple_upgrade.pipeline

```diff
@@ -152,6 +152,10 @@ node('onmetal-provisioner') {
     osa.run_persistent_resources_tests('verify', 'after_upgrade')
     osa.run_persistent_resources_tests('cleanup')
     
+    // Pase persistent resource results
+    osa.setup_parse_persistent_resources()
+    osa.parse_persistent_resources_tests()
```

Comment:
Yes it will include create,verify,cleanup, before_upgrade and after_upgrade -- give you idea it will produce a json like such: http://paste.openstack.org/show/587674/ 

If the pipeline fails during the upgrade I was under the impression that the entire upgrading process will stop and jump to cleaning up the on metal, which will result in my parsing not even being called.